docs/modules/general/introduction.mdx

```diff
@@ -27,8 +27,8 @@ As of now, there are over 1000 maps made for PGM!
 ### How it Works
 
 All maps made for PGM are configured using a unique XML file called `map.xml`.
-XML is structured and layered similar to HTML, making it easily readable.
-PGM reads `map.xml` when the server is loaded, and when the match starts it applies all the functions that is defined in the XML file onto the game.
+XML is structured and layered like HTML, making it easily readable.
```

Comment:
keep "similar to", it is not like html, html supports having unclosed elements and a bunch of weird tricks

Reply:
Fixed it back